Maxfish2

GetHubMax opened the pull request:
Should remove Building UI when focus on a fish. Down the line I want t animate move off.

GetHubMax committed:
Modes of view, shows only usual UI in the right mode

GetHubMax committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into maxFish2

# Conflicts:
#	Assets/MainGame.unity
#	Assets/Prefabs/GameObjects/SailfinTang.prefab

GetHubMax committed:
Working on AI for fish to eat at reef. A random fish and random reef will be selected in gamescipt.
The fish will be give the target reef and then the fish will move to it when it as a cleer line.
I'm using 'MoveRandomPath'as the base for the AI.

Sam-Costigan commented:
Sweet man looks good. I'm just wondering what all the filename changes are about? 124 files changed seems like a lot for this. We probably don't need the `_BACKUP_`, `_BASE_` etc. files either, it seems like those were created by the merge conflict tool you were using.

GetHubMax committed:
Smarter Fish in the works

Build a system of modes  for the fish to be in. Using ray cast to check
if there is anything in the way when moving to reef.
